test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl

```
@@ -173,6 +173,7 @@ inputs = {
   helm_exporter_github_owner        = "dfds"
   helm_exporter_repo_name           = "platform-manifests-qa"
   helm_exporter_repo_branch         = "main"
+  helm_exporter_target_namespaces   = "fluentd, flux-system, monitoring, traefik-green-variant"
```

```suggestion
  helm_exporter_target_namespaces   = "fluentd,flux-system,monitoring,traefik-green-variant"
```

I am not sure if the space can be handled properly, but I'm pretty sure without the space it should work properly.